websitetoolbox.com.apexforum.json

```diff
@@ -0,0 +1,36 @@
+{
+  "providerId":"WebsiteToolbox.com",
+  "providerName":"Website Toolbox",
+  "serviceId":"apexforum",
+  "serviceName":"Website Toolbox Forum",
+  "version": 1,
+  "syncPubKeyDomain" : "websitetoolbox.com",
+  "logoUrl":"https://cdn.websitetoolbox.com/images/logos/wt_logo_transparent.png",
+  "description":"Enables the root of a domain to work with Website Toolbox.",
+  "variableDescription":"",
+  "records":[
+    {
+      "type":"REDIR301",
+      "host": "@",
+      "target":"https://www.%fqdn%"
+    },
+    {
+      "type":"CNAME",
+      "host":"www",
+      "pointsTo":"custom.websitetoolbox.com",
+      "ttl":3600
+    },
+    {
+      "type": "TXT",
```

Comment:
For an SPF record you should use the SPFM type.  This will allow multiple templates that try to set SPF records to co-exist.